### PR TITLE
Ben_NOAA_changes

### PR DESCRIPTION
**Pull request by qq23840**
Added a README.txt for the output zipfile. Not sure whether the code to copy it into the zip directory will actually work. 
Added AGAGE data availability/usage statements as attributes to the output files, as per NOAA ObsPack 
Added a json converting standard_name attributes of the mf and mf_repeatability variables to accepted standard_names under CF conventions
NOT YET added a test for CF Compliance (Gareth and I to discuss)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -133,4 +133,5 @@ dmypy.json
 .vscode
 .DS_store
 ~$*
-agage-gdrive.json
\ No newline at end of file
+agage-gdrive.json
+ben_*
```
Comment (lukewestern) [inline .gitignore:137]: This shouldn't be in .gitignore for everyone. Better to just not add those files locally.

**Comment by lukewestern** [comment]
I would make the ZIPFILE_README.txt file a markdown (.md) file and remove all the hashes on the left side as they serve no purpose.
The naming of the file could also be more informative, e.g., agage-data_README.md or something similar. (zipfle isn't obvious)

**Comment by lukewestern** [comment]
Also, is it worth putting a licence in the README? And/or should it appear as an attribute?

CC BY 4.0 (or maybe even CC BY-SA 4.0) I would imagine

**Comment by qq23840** [comment]
Made and pushed the changes you mentioned @lukewestern - should be ready to go now I think

File: .gitignore [hunk 2]
```diff
@@ -132,5 +132,4 @@ dmypy.json
 **/config.yaml
 .vscode
 .DS_store
-~$*
-agage-gdrive.json
```
Comment (lukewestern) [inline .gitignore:136]: I think this should be in the .gitignore as we wouldn't want it to go public

File: agage_archive/run.py [hunk 3]
```diff
@@ -268,6 +268,14 @@ def run_all(network,
         with ZipFile(out_pth, "w") as f:
             pass
 
+    # Add the README file to the output .zip directory
+    
+    readme_path = data_file_path(filename='ZIPFILE_README.txt',
```
Comment (lukewestern) [inline agage_archive/run.py:273]: This filename has changed in your most recent commits, no?

**Comment by lukewestern** [comment]
It says that there are merge conflicts in test_formatting.py then need to be fixed @qq23840. 
These will need to be fixed before merge.

**Comment by qq23840** [comment]
Merge conflicts fixed - was just a silly formatting thing. Also committed the changes in your previous comments.

**Comment by mrghg** [comment]
Thanks for doing this Ben. Please take a look at my latest fix to test_formatting, and note that I don't think the standard_names table was actually being read before?

The CF test still fails for me, because it doesn't seem to like the repeatability variable as having a standard name. Should we prevent these variables from having a standard name (i.e., only have the "mf" variable have a standard name)?

**Comment by mrghg** [comment]
@qq23840 or @lukewestern , are either of you available to review the latest changes I've made? See commit messages

**Comment by qq23840** [comment]
I can take a look now

**Pull request merged**